user_tools/src/spark_rapids_pytools/resources/emr-configs.json

```diff
@@ -234,5 +234,8 @@
         }
       }
     }
+  },
+  "clusterSpecs": {
+    "minWorkerNodes": 2
```

Comment:
do you need to add gpuScaleFactor here as well?

Reply:
I put a default in the qualification-config.
My thought is that I did not want to put a factor in a platform until it is confirmed and benchmarked. Should we go for 0.8 in EMR too?

Reply:
sure, that's fine for now